server/init/eventConfig.js

```diff
@@ -0,0 +1,20 @@
+import moment from 'moment-timezone';
+
+const config = {
+  cfpStateDate: "2018-04-25",
+  cfpEndDate: "2018-05-31",
+  votingStartDate: "2018-06-01",
+  votingEndDate: "2018-06-10",
+}
+
+const cfp = moment().tz('Israel').isBetween(config.cfpStateDate, config.cfpEndDate, 'day', "[]");
+const voting = moment().tz('Israel').isBetween(config.votingStartDate, config.votingEndDate, 'day', "[]");
```

Comment:
so these two fields get initialized when the server starts, right? 
so they won't get updated over time unless the container is restarted, which is something you don't control in heroku as far as I know (and anyway it doesn't seem to be a good thing to rely on)

Reply:
good catch